### ISSUE TITLE
tox.ini ignore list not recognized -- but works via CLI

### ISSUE THREAD

**Issue by pydolan** [AtomLinter/linter-flake8#622]
For some reason, my ignored flake8 errors in tox.ini are not recognized by the linter, and the errors are reported in Atom. I'm using linter-ui-default to see these. If I run `flake8` from the command line on a file where an ignored error is occurring, it properly ignores the errors I want to ignore (and shows others that it should).

Any thoughts on what could be the problem or how I can debug this further? Thanks!

**Comment by Arcanemagus** [comment]
Where is the `tox.ini` located with regards to the project as Atom is opened to? If you open Atom's Developer Tools and place a breakpoint here, the `configFilePath` variable should be set to where it _thinks_ the config file is (or isn't).
https://github.com/AtomLinter/linter-flake8/blob/0f3d35965a0761c6cb229ebc7a1d3938215376a2/lib/main.js#L202

**Comment by Arcanemagus** [comment]
Hmmmmm, if you add this to the file somewhere, reload Atom to catch the changes, and place a breakpoint after it to read the value, does the output match what you see when you run `flake8 --bug-report` in your shell?

```js
bugReport = await helpers.exec(execPath, [ '--bug-report' ], { cwd: baseDir });
```

**Comment by Arcanemagus** [comment]
> some unicode characters

Ah ha! unfortunately on a lot of systems the processes executed through Atom are ran in the `C` locale, which isn't able to understand those characters. Your terminal is probably using a proper `en_US.UTF-8` which is why it's working there.

There are a few other issues related to that open, solving them will solve this. Thanks for the confirmation!
